manifests/quick-start-minimal.yaml

```diff
@@ -908,6 +908,7 @@ rules:
   - workflows/finalizers
   - workflowtasksets
   - workflowtasksets/finalizers
+  - artifactgctasks
```

Comment:
For consistency`workflowartifactgctaskset`

Reply:
do you like "workflowartifactgctaskset" better than "workflowartifactgctask"? I was thinking this was more of a task than a task set.? (WorkflowTaskSet, on the other hand, actually defines a list of Tasks)

Reply:
changed to "workflowartifactgctaskset"

Reply:
changed to _workflowartifactgctaskset_

Reply:
back to _workflowartifactgctask_ since you said you were open to that in a separate conversation (thank you)